drinfomon.lic

```diff
@@ -588,7 +605,7 @@ vault_titles = {
   "Ratha"              => ["[[Ratha, Carousel Square]]"],
   "Riverhaven"         => ["[[Riverhaven, Carousel Chamber]]"],
   "Shard"              => ["[[Shard, Carousel Chamber]]"],
-  "Therenborough"      => ["[[Therenborough, Carousel Chamber]]"]
+  "Therenborough"      => ["[[Therenborough, Carousel Square]]"]
```

Comment:
Did not mean to undo https://github.com/rpherbig/dr-scripts/commit/959e4f1b1fe5f612b23c89a04a4f6dcaf8501a0e
```suggestion
  "Therenborough"      => ["[[Therenborough, Carousel Chamber]]"]
```